hoomd/hpmc/update.py

```diff
@@ -20,8 +20,8 @@ class BoxMC(Updater):
 
     Args:
         betaP (`float` or :py:mod:`hoomd.variant.Variant`):
-            :math:`\frac{p}{k_{\mathrm{B}}T}` (units of inverse area in 2D or
-            inverse volume in 3D).
+            :math:`\frac{p}{k_{\mathrm{B}}T}` :math:`[\mathrm{area}^{-1}]`
+            in 2D or :math:`[\mathrm{volume}^{-1}]` in 3D.
         trigger (hoomd.trigger.Trigger): Select the timesteps to perform box
             trial moves.
 
```

Comment:
The delta key in the volume dict for this class has written that the units are area and volume, but we should put that in the new style of writing units.